cou/apps/app.py

```diff
@@ -128,6 +129,27 @@ def __str__(self) -> str:
             yaml.dump(summary, stream)
             return stream.getvalue()
 
+    @property
+    def series(self) -> str:
+        """Ubuntu series of the application.
+
+        :return: Ubuntu series of application. E.g: focal
+        :rtype: str
+        """
+        return self.status.series
+
+    @property
+    def current_os_release(self) -> Optional[OpenStackRelease]:
+        """Current OpenStack Release of the application.
+
+        :return: OpenStackRelease object
+        :rtype: OpenStackRelease
+        """
+        os_versions = {unit_values.get("os_version") for unit_values in self.units.values()}
+        if len(os_versions) == 1:
+            return list(os_versions)[0]
+        return None
```

Comment:
not-blocking: Should we provide some warning logs if there are two different os_versions? Returns `None` make sense in that case, but the logs would be nice.

Reply:
Warning logs added